static/app/views/settings/project/projectKeys/projectKeyCredentials.tsx

```diff
@@ -192,6 +192,19 @@ function ProjectKeyCredentials({
           </TextCopyInput>
         </FieldGroup>
       )}
+
+      {data.useCase && (
+        <FieldGroup
+          label={t('Use Case')}
+          help={t('Whether the DSN is for the user or for internal data submissions.')}
+          inline
+          flexibleControlStateSize
+        >
+          <StyledField label={null} inline={false} flexibleControlStateSize>
+            {data.useCase}
+          </StyledField>
```

Comment:
There's probably prettier ways to display a single string. Open to suggestions here!